plugins/Morpheus/stylesheets/less/_variables.less

```diff
@@ -17,6 +17,8 @@
 @theme-color-background-highContrast:  #202020;
 @theme-color-base-series:              #ee3024;
 
+@theme-color-border: @color-silver-l80;
```

Comment:
I noticed that we have that border and even more with all the redesigns (the list component, the tables, the forms)… it's useful to have a variable for that instead of every time trying to find the correct color again (happened to me a lot lately, it's honestly a real pain).